app/models/maintenance_tasks/task_data.rb

```diff
@@ -157,6 +170,11 @@ def new
       MaintenanceTasks::Task.named(name).new
     end
 
+    # @return [Boolean] whether the Task has any Run.
```

Comment:
Maybe we could call it `ran?` or `has_any_run?`. We can also make this `@api private` because it shouldn't really be used beyond `find` ?